custom_components/tuya_local/devices/cecotec_bigdry_4000_dehumidifier.yaml

```diff
@@ -54,6 +54,7 @@ entities:
         unit: C
   - entity: switch
     translation_key: uv_sterilization
+    # Not supported by the device, still shown by Tuya
```

Comment:
It may be better to move this to an attribute rather than having an entity that does not work as advertised